web/index.html

```diff
@@ -552,64 +601,147 @@ <h3>Used in</h3>
                     };
 
                     const animateScroll = (currentTime) => {
+                        if (this.isScrolling) {
+                            return;
+                        }
                         const timeElapsed = currentTime - startTime;
-                        const run = easeInOutQuad(timeElapsed, start, position - start, duration);
-
+                        const run = easeInOutQuad(timeElapsed, start, 0 - start, duration);
                         window.scrollTo(0, run);
-
                         if (timeElapsed < duration) requestAnimationFrame(animateScroll);
                     };
 
                     requestAnimationFrame(animateScroll);
                 },
                 clearSaveData: function () {
-                    this.saveData = [];
+                    this.saveData = new Set();
+                    this.saveDataArray = [];
                     localStorage.setItem("saveData", "");
+                    window.location.reload();
```

Comment:
Clearing the save data should reload the page

Reply:
Because without refreshing, all of the already loaded chunks will still be sorted based on their previously saved data, and the current page just has a few other oddities. Reloading only takes a second, makes sure everything is cleaned up nicely, and doesn't really have any negative set effects

Reply:
Oof sounds like bad vue state to me, is chunk sorting being done imperatively? Can we make more things computed?